db/migrate/20150714144500_review_discovery_permissions.rb

```diff
@@ -0,0 +1,16 @@
+class ReviewDiscoveryPermissions < ActiveRecord::Migration
+  def up
+    if mgr = Role.find_by_name("Discovery Manager")
```

Comment:
just a nit, it's a nice convention to surround assignment by brackets if it's part of `if` so everyone knows that there's not missing second equals sign

`if (mgr = Role.find_by_name("Discovery Manager"))`